### PR TITLE
Update important-upgrade-notes.rst

### PR DESCRIPTION
**Pull request by amyblais**
Also notify Gitlab and add to changelog

### CODE REVIEW

File: source/administration/important-upgrade-notes.rst [hunk 1]
```diff
@@ -16,6 +16,12 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v5.35.0                                            | v5.35.0 introduces a new feature to search for files. Search results for files shared in the past may be incomplete until a                                      |
```
Comment (esethna) [inline source/administration/important-upgrade-notes.rst:19]: v5.35.0 introduces a new feature to search for files. Search results for files shared in the past may be incomplete until a content extraction command is executed to extract and index the content of files already in the database. Instances running Elasticsearch or Bleve search backends will also need to execute a Bulk Indexing after the content extraction is complete. Please see more details in this blog post.

**Review by esethna — CHANGES_REQUESTED** [review]

**Review by esethna — APPROVED** [review]
Thanks!

File: source/administration/important-upgrade-notes.rst [hunk 2]
```diff
@@ -16,6 +16,11 @@ Important Upgrade Notes
...
```
Comment (cwarnermm) [inline source/administration/important-upgrade-notes.rst:19]:
```suggestion
| v5.35.0                                            | v5.35.0 introduces a new feature to search for the contents of files shared in Mattermost. Search results for file contents shared prior to upgrading to v5.35.0 may be incomplete until a                                      |
```

**Review by cwarnermm — COMMENTED** [review]
Suggestions provided inline. The key feedback is that the new file search feature specifically searches on file contents, not the files themselves.

File: source/administration/important-upgrade-notes.rst (hunk 2 above)
Comment (amyblais) [reply source/administration/important-upgrade-notes.rst:19]: @esethna Please help confirm ^

**Review by esethna — APPROVED** [review]

**Pull request merged**